charts/dremio-module/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: dremio-module-multi-chart
+description: A Helm chart for installing a module for Fybrik in Kubernetes
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: "0.0.0"
+dependencies:
+- name: dremio-chart
+  version: "0.0.0"
+  repository: "file://../dremio-chart"
```

Comment:
we don't have dremio-chart now

Reply:
The `dremio-cluster` chart has the name `dremio-chart` (in the file `charts/dremio-cluster/Chart.yaml`). Maybe it is better to change the name of the chart to `dremio-cluster`?

Reply:
Changed the structure + names